spec/src/main/asciidoc/messaging-spec.adoc

```diff
@@ -1245,18 +1241,18 @@ properties are undefined.
 
 ==== Provider-specific properties
 
-JMS reserves the ‘JMS___<vendor_name>__’ property name prefix for
+Jakarta Messaging reserves the ‘JMS___<vendor_name>__’ property name prefix for
```

Comment:
There is a problem here with the underscore missing between JMS and <vendor_name>. The missing underscore is showing up on the next line
This is formatting issue.

![image](https://user-images.githubusercontent.com/16149023/81455477-f5a73580-91ac-11ea-9d67-79c9e2fe1d17.png)

![image](https://user-images.githubusercontent.com/16149023/81455530-26876a80-91ad-11ea-8082-3bab45c3d23e.png)

Reply:
Fixed.